pom.xml

```diff
@@ -311,6 +311,13 @@
                 <groupId>com.squareup.okhttp3</groupId>
                 <artifactId>okhttp</artifactId>
                 <version>${okhttp.version}</version>
+                <!-- Library with known vulnerability and no fixed version -->
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.jetbrains.kotlin</groupId>
+                        <artifactId>kotlin-stdlib</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
Does this actually work? Not sure this would exclude `kotlin-stdlib` from the dist, and also not sure if `okhttp` will actually work without this dependency included.

Another option is to get rid of `okhttp` completely, which I think is only needed due to `[openshift-restclient-java](https://github.com/openshift/openshift-restclient-java)`, which we need to get rid of anyways as that's no longer maintained. The placement would be https://github.com/fabric8io/kubernetes-client/, which has integration with many different http clients, including plain `jdk`.

Reply:
@vmuzikar @pedroigor FIY

Reply:
+1 if we can get rid of openshift-restclient-java and answering your question, yes it works. Below you can find a side by side comparison:

- main branch
```
❯ mvn dependency:tree | grep kotlin-stdlib
[INFO] |  |  |  +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO] |  |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO] |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
[INFO] |  |  \- org.jetbrains.kotlin:kotlin-stdlib:jar:1.7.20:compile
[INFO] |  |  |  |  +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO] |  |  |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO] |  |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
[INFO] |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib:jar:1.7.20:compile
[INFO]    |  |  |  +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO]    |  |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO]    |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
[INFO]    |  |  \- org.jetbrains.kotlin:kotlin-stdlib:jar:1.7.20:compile
[INFO] |  |  |  |  +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO] |  |  |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO] |  |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
[INFO] |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib:jar:1.7.20:compile
```

- Outcome from the changes proposed here

```
❯ mvn dependency:tree | grep kotlin-stdlib
[INFO] |  |     +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO] |  |     |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO] |  |     \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
[INFO] |  |  |     +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO] |  |  |     |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO] |  |  |     \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
[INFO]    |  |     +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO]    |  |     |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO]    |  |     \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
[INFO] |  |  |     +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.20:compile
[INFO] |  |  |     |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.7.20:compile
[INFO] |  |  |     \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.7.20:compile
```